.github/workflows/static-checks.yml

```diff
@@ -156,7 +185,15 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: '17'
-          cache: 'maven'
+
+      # restore maven dependencies only. 'depsonly' keys are safe to restore
+      # since they don't contain any artifacts created by mvn install
+      - uses: actions/cache@v3
```

Comment:
shouldn't this be `actions/cache/restore@v3` ?

Reply:
`actions/cache@v3` is a shorthand to do both a restore and save once the job succeeds.

Reply:
it does run maven as part of `${MVN} test -pl 'web-console'`, but that might not pull most dependencies, so I'm fine switching to restore-only if we are afraid to clobber the cache with fewer dependencies.